### DISCUSSION TITLE
希望deepin & UOS把源管理放进控制中心

### DISCUSSION THREAD

**Discussion by widon1104** [linuxdeepin/developer-center#2938] [Features Request & Ideas | 特性请求 & 头脑风暴]
## 缺陷描述

有时候安装一个软件过程中，软件就会往/etc/apt/sourse.list.d/里面放进软件的源信息，有些软件源信息是国外的很慢不说有时候还会被墙，导致用户升级软件显示失败。
希望把源管理放进控制中心，功能大概就是可以默认激活源、删除源，不删除关闭源等功能，这样对源的管理就更加完善了。

### 发行版以及版本

V20

**Comment by XMuli** [comment]
以前 v15 是在控制中心是有智能源功能的，后移除了， 现在已经着手在社区版回复该功能了

**Comment by Darki-luo** [comment]
> 以前 v15 是在控制中心是有智能源功能的，后移除了， 现在已经着手在社区版回复该功能了

软件的源会像以前那样更新吗？高校和国内的源好像还是15的

**Comment by XMuli** [comment]
@Darki-luo 会的， 我们镜像源到时候也会同步的（不然没有意义）

**Comment by BLumia** [comment]
这个目前版本应该已经加回去了。

![image](https://user-images.githubusercontent.com/10095765/160571616-cd787124-ec0b-4980-810a-c7a495d0174f.png)
